platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/reconnect/ReconnectLearnerThrottle.java

```diff
@@ -95,15 +95,15 @@ private void killNodeIfThresholdMet() {
      * Check if a reconnect is currently allowed. If not then kill the node.
      */
     public void exitIfReconnectIsDisabled() {
-        if (!settings.isActive()) {
+        if (!config.active()) {
             logger.warn(STARTUP.getMarker(), () -> new UnableToReconnectPayload(
                             "Node has fallen behind, reconnect is disabled, will die", selfId.getIdAsInt())
                     .toString());
             SystemUtils.exitSystem(SystemExitReason.BEHIND_RECONNECT_DISABLED);
         }
 
-        if (settings.getReconnectWindowSeconds() >= 0
-                && settings.getReconnectWindowSeconds()
+        if (config.reconnectWindowSeconds() >= 0
```

Comment:
Since we support Duration as type we should create an issue to change reconnectWindowSeconds to Duration.

Reply:
can you create an issue for that?

Reply:
https://github.com/hashgraph/hedera-services/issues/6280